### PR TITLE
Mb/41 test strategy

### PR DESCRIPTION
**Pull request by MartinBruland**
**DONE**
- Added Cypress as testing framework. 
- Added test: Page navigation.

**TODO**
- Define what to test.
- Implement defined tests.

### CODE REVIEW

**Comment by boyum** [comment]
Cool! As a separate PR, can we add this to CI? It would be awesome if it was run on each PR.

**Comment by boyum** [comment]
Why is the `tsconfig` file empty? Could we extend from web/tsconfig.json?

**Comment by MartinBruland** [comment]
> Why is the `tsconfig` file empty? Could we extend from web/tsconfig.json?

Received the following error when 'tsconfig' file was missing:  "TS18002: The 'files' list in config file 'tsconfig.json' is empty."
Added it as a quick fix.

**Review by boyum — APPROVED** [review]

File: .github/workflows/validate.yml [hunk 1]
```diff
@@ -47,6 +47,21 @@ jobs:
...
         working-directory: ${{ env.working-directory }}
 
+  cypress-run:
+    name: Run Cypress Tests
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
```
Comment (boyum) [inline .github/workflows/validate.yml]:
```suggestion
        uses: actions/checkout@v3
```

**Review by boyum — APPROVED** [review]

**Pull request closed without merging**